llvm/lib/Transforms/Instrumentation/PGOInstrumentation.cpp

```diff
@@ -2015,9 +2053,8 @@ static bool annotateAllFunctions(
 
   // If the profile marked as always instrument the entry BB, do the
   // same. Note this can be overwritten by the internal option in CFGMST.h
-  bool InstrumentFuncEntry = PGOReader->instrEntryBBEnabled();
-  if (PGOInstrumentEntry.getNumOccurrences() > 0)
```

Comment:
I think by removing this you have changed the behavior of the code? With the old version, I think if the user explicitly specified `-pgo-instrument-entry=false` it would override instrEntryBBEnabled ? At least I believe that is the behavior when checking `if (PGOInstrumentEntry.getNumOccurrences() > 0)`.